ArduSub/system.cpp

```diff
@@ -157,6 +157,8 @@ void Sub::init_ardupilot()
     g2.actuators.initialize_actuators();
 
     // flag that initialisation has completed
+    update_actuators_from_jsbuttons();
```

Comment:
This should happen before the actuators initialise, right?

Reply:
YES!